openshift2nulecule/cli/main.py

```diff
@@ -82,7 +86,7 @@ def run(self):
             logger.critical(msg)
             raise Exception(msg)
 
-        if args.export_images != 'none' and not args.registry_host:
+        if not args.skip_push and args.export_images != 'none' and not args.registry_host:
             msg = "With --export-images you need also set --registry-host"
```

Comment:
If the above message is improved to reflect the additional option of `--skip-push`.
Or
We can add additional nested `if` to see if the `--skip-option` is set and give a message according to that.